packages/react-bootstrap-table2-example/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-bootstrap-table2-example",
-  "version": "0.1.10",
+  "version": "0.1.11",
```

Comment:
Don''t patch version manually, we use `lerna`. Please remove this changes

Reply:
Actually, I just rebased branch to develop, i.e. i didn't patch it manually. it came from develop branch, I think